### PR TITLE
update-check: look into version directories

### PR DESCRIPTION
**Pull request by Chocimier**
This find updates of software from xfce, mate, apache, gimp, videolan, rest of kde and some individual like batman-adv, hatari, oidentd, wine-mono.

### CODE REVIEW

File: common/xbps-src/shutils/update_check.sh [hunk 1]
```diff
@@ -36,15 +36,19 @@ update_check() {
         rx=
         urlpfx="${url}"
         urlsfx=
+        dirpfx=
         case "$url" in
```
[the remark this answers is not in the document]
Comment (Chocimier) [reply common/xbps-src/shutils/update_check.sh]: There are two branches. `*)`-one can be moved bottom of `case` as the other stops execution, but is it really cleaner?

[the remark this answers is not in the document]
Comment (pullmoll) [reply common/xbps-src/shutils/update_check.sh]: I didn't and don't see a second branch, if I look at the entire file after the patch. I may be not seeing it for whatever reason.

[the remark this answers is not in the document]
Comment (Chocimier) [reply common/xbps-src/shutils/update_check.sh]: In github webpage head of git branch is 0b2d173 that has more code. I guess you are previewing pr with other tool?

[the remark this answers is not in the document]
Comment (pullmoll) [reply common/xbps-src/shutils/update_check.sh]: D'OH! Never mind. You didn't squash the commits, that's why I didn't see it.

**Pull request merged**